rd_ui/app/scripts/controllers/alerts.js

```diff
@@ -120,6 +122,40 @@
       },
       controller: function ($scope) {
         $scope.subscribers = AlertSubscription.query({alertId: $scope.alertId});
+        $scope.subscription = {};
+        $scope.destinations = Destination.query();
+
+        $scope.destinationsDisplay = function(destination) {
+            var icons = {
+                'slack': 'fa-slack',
+                'email': 'fa-envelope',
```

Comment:
Just for clarification -- do you mean include icon on the model in the backend that gets sent as part of the API response?